generatebundlefile/hack/pullpush.sh

```diff
@@ -83,5 +83,13 @@ helm pull oci://646717423341.dkr.ecr.us-west-2.amazonaws.com/jetstack/cert-manag
 helm push cert-manager-v1.5.3-4ae27c6a1df646736d9e276358d0a6b2daf99f55.tgz "oci://$local_ecr_public"
 # sha256:a507b9e9e739f6a2363b8739b1ad8f801b67768578867b9fae7d303f9e8918e8
 
+# Populate test cases
 helm pull oci://public.ecr.aws/l0g8r8j6/eks-anywhere-test --version v0.1.1-4280284ae5696ef42fd2a890d083b88f75d4978a-helm
-helm push eks-anywhere-test-v0.1.1-4280284ae5696ef42fd2a890d083b88f75d4978a-helm.tgz "oci://$local_ecr_public"
+mv eks-anywhere-test-v0.1.1-4280284ae5696ef42fd2a890d083b88f75d4978a-helm.tgz eks-anywhere-test-v1.0.1-helm.tgz
```

Comment:
We have to change the version of helm charts to `1.0.1` (no leading `v`, no trailing `-prerelease`)

Reply:
Same thing fixed here https://github.com/aws/eks-anywhere-build-tooling/pull/557/commits/a771eaa0ae98ba09b85ff0b8953c32d6f8b9fbc4

Reply:
I guess this is a hack file, but this is super brittle, and hopefully not part of any of our pipelines?

Reply:
when we build it, it would not have the `v` in it because it gets that from `release` in the Chart.yaml, so I wouldn't expect this to work. https://github.com/aws/eks-anywhere-build-tooling/pull/585 merged, so the `v` situation is taken care of.